core/src/commonMain/kotlin/entity/Member.kt

```diff
@@ -42,7 +42,7 @@ public class Member(
     /**
      * The member's effective name, prioritizing [nickname] over [globalName] and [username].
      */
-    public val effectiveName: String get() = nickname ?: (this as User).displayName
+    public val effectiveName: String get() = nickname ?: (this as User).effectiveName
 
```

Comment:
Can't we replace that with a super call?

Reply:
no, super only works for members, it isn't a normal expression